src/domain/place/placeText/PlaceText.tsx

```diff
@@ -1,20 +1,26 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 
 import { usePlaceQuery } from '../../../generated/graphql';
 import useLocale from '../../../hooks/useLocale';
 import getLocalisedString from '../../../utils/getLocalisedString';
 
 interface Props {
   placeId: string;
+  showErrorText?: boolean;
 }
 
-const PlaceText: React.FC<Props> = ({ placeId }) => {
+const PlaceText: React.FC<Props> = ({ placeId, showErrorText = false }) => {
   const locale = useLocale();
+  const { t } = useTranslation();
   const { data } = usePlaceQuery({
     variables: { id: placeId },
   });
+  const text = getLocalisedString(data?.place?.name || {}, locale);
+  const errorText =
+    showErrorText && placeId && !text ? t('placeText.noPlaceFoundError') : '';
 
-  return <>{getLocalisedString(data?.place?.name || {}, locale)}</>;
+  return <>{text || errorText}</>;
```

Comment:
This feels unnecessarily complicated. Maybe could get rid of `!text` because it is redundant with this code.

Reply:
I also think that `t('placeText.noPlaceFoundError')` shouldn't be in this component but rather provided as errorText prop

Reply:
Because this is general reusable component and text "Paikka on valittu..." is specific to UnitPlaceSelector.